package.json

```diff
@@ -64,11 +65,6 @@
             "plugin:@typescript-eslint/recommended"
         ]
     },
-    "husky": {
-        "hooks": {
-            "pre-push": "pretty-quick --staged && yarn lint && yarn tsc && yarn test --watchAll=false"
-        }
-    },
```

Comment:
why remove hooks?

Reply:
> I've removed the pre-push hook for the time being because I was finding it painful to work with

They were making things a bit difficult for me, but I can turn them back on if you think that would be preferable?

I personally prefer to use CI for this, but I'm also happy to go with the consensus if it makes it easier to work across projects.